cmd/samples/pso/activities.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"context"
+	"math/rand"
+	"time"
+
+	"go.uber.org/cadence/activity"
+	"go.uber.org/zap"
+)
+
+/**
+ * Sample activities used by file processing sample workflow.
+ */
+const (
+	initParticleActivityName   = "initParticleActivityName"
+	updateParticleActivityName = "updateParticleActivityName"
+)
+
+var rng *rand.Rand
+
+// This is registration process where you register all your activity handlers.
+func init() {
+	// initialize the RNG
+	// WARNING: the randomness of activity scheduling with multiple workers makes random number generation truly random and not repeatable in debugging
+	rng = rand.New(rand.NewSource(time.Now().UnixNano()))
```

Comment:
Add comment that worker.ReplayWorkflowHistoryFromJSONFile should be used to troubleshoot a specific workflow failure.

Reply:
Cool, I didn't know about that feature